app/lib/ostatus/atom_serializer.rb

```diff
@@ -364,8 +364,6 @@ def serialize_status_attributes(entry, status)
       append_element(entry, 'category', nil, term: tag.name)
     end
 
-    append_element(entry, 'category', nil, term: 'nsfw') if status.sensitive?
```

Comment:
I *think*, reading #7011 closely, the logic that's desired is `if status.sensitive? && status.with_media?`

I would prefer that instead of removing it altogether, we keep it like this for compatibility's sake. I agree that NSFW is not the term we *want* to be using, but harm reduction is more important then ideological purity, and it's the term that part of the fediverse has standardized on.